Engine/DataFeeds/LiveTradingDataFeed.cs

```diff
@@ -539,6 +577,44 @@ private Subscription CreateUniverseSubscription(SubscriptionRequest request)
                 });
                 enumerator = enqueable;
             }
+            else if (request.Universe is OptionChainUniverse)
```

Comment:
See my comments above. I believe that such sprawling conditionals are good sign for timely (not premature) refactoring. Added to todos.